src/main/kotlin/nodes/AVLNode.kt

```diff
@@ -0,0 +1,7 @@
+package nodes
+
+import primitives.Node
+
+class AVLNode<K : Comparable<K>, V>(k: K, v: V) : Node<K, V, AVLNode<K, V>>(k, v) {
```

Comment:
возможно, стоит использовать более полные название для переменных (хотя практикуется и ваш вариант)

```suggestion
class AVLNode<K : Comparable<K>, V>(key: K, value: V) : Node<K, V, AVLNode<K, V>>(key, value) {
```